Create ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md (#1964)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1019-20-8-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md

---
date: ""

> title: Disable dashboard availability select if user has no permission to request roles to populate it. (#6021) [GRAFT][release/cd] (#6173)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Disable dashboard availability select for users without permission to request roles
```

> title: Disable dashboard availability select if user has no permission to request roles to populate it. (#6021) [GRAFT][release/cd] (#6173)

Reviewer: ```suggestion
title: Dashboard availability selection is hidden for users who do not have the required permission
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-57128
version: 1019.20.8
---

> Disable dashboard availability select if user has no permission to request roles to populate it. (#6021) [GRAFT][release/cd] (#6173)

Reviewer: ```suggestion
Previously, the dashboard availability selection was always visible for users, even if they did not have the permission to see it. With this change, the dashboard availability selection is hidden for users who do not have the required permission. Users must have User Management READ permission to see and modify the availability property of a dashboard.
```

> Disable dashboard availability select if user has no permission to request roles to populate it. (#6021) [GRAFT][release/cd] (#6173)

Author: The Claude AI suggested to use the following body:
```suggestion
In the past, the dashboard availability select was always enabled for users, even if they did not have permission to request the roles required to populate it. With this change, the dashboard availability select is now disabled for users who do not have permission to request roles. This improves the user experience by avoiding confusion and clearly indicating to users when they are unable to use the dashboard availability select feature due to insufficient permissions.
```
